File: 0026. Remove Duplicates from Sorted Array/Solution.py
class Solution:
    def removeDuplicates(self, nums):
        """
        type nums: List[int]
        rtype: int
        """
        l = 0
        for x in range(l + 1, len(nums)):
            if nums[x] != nums[l]:
                l += 1
                nums[l] = nums[x]
        return l + 1

File: 0026. Remove Duplicates from Sorted Array/Solution.cpp
#include <vector>

using std::vector;

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int l = 0;
        for (int i = l + 1; i < nums.size(); i++)
        {
            if (nums[l] != nums[i])
            {
                l++;
                nums[l] = nums[i];
            }
        }
        return l + 1;
    }
};